src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,28 @@
+package duke.command;
+
+import duke.exceptions.DukeException;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents a command that shows the task in the task list.
+ */
+public class ListCommand extends UserCommand {
+
+    /**
+     * @param userInput user's input.
+     */
+    public ListCommand(String userInput) {
+        super(userInput);
+    }
+
+    /**
+     * @param taskList task list containing all the tasks.
+     * @param ui       ui that prints output.
```

Comment:
Should the abundant spaces between the parameter and its description be removed? I notice the same issue at other places too.